docusaurus.footer.config.js

```diff
@@ -0,0 +1,53 @@
+                                </svg>
+                            </a>
+                            <a href="/community/#discussions" title="matrix" target="__blank">
+                                <svg viewBox='0 0 16 16' width="16" height="16" xmlns='http://www.w3.org/2000/svg'>
+                                    <path d='M0.421333 0.366667V15.6333H1.52V16H0V0H1.52V0.366667H0.421333ZM5.11667 5.20667V5.978H5.13867C5.3308 5.69595 5.58569 5.46228 5.88333 5.29533C6.172 5.132 6.50733 5.052 6.88333 5.052C7.24333 5.052 7.572 5.12333 7.87067 5.26133C8.16933 5.4 8.394 5.64933 8.55067 6C8.72 5.75067 8.95067 5.52933 9.24 5.33867C9.52933 5.14733 9.87333 5.052 10.2707 5.052C10.5727 5.052 10.852 5.08933 11.1107 5.16333C11.3693 5.23667 11.588 5.354 11.7727 5.51667C11.9567 5.68 12.0987 5.88933 12.2033 6.15067C12.3047 6.412 12.3567 6.726 12.3567 7.09533V10.914H10.7907V7.68C10.7907 7.48933 10.784 7.30733 10.7693 7.13867C10.7586 6.98555 10.7178 6.83604 10.6493 6.69867C10.5828 6.57305 10.4814 6.46933 10.3573 6.4C10.228 6.32667 10.0527 6.28933 9.834 6.28933C9.61267 6.28933 9.434 6.332 9.29867 6.41533C9.16546 6.49723 9.05533 6.61172 8.97867 6.748C8.89839 6.89151 8.84613 7.04897 8.82467 7.212C8.79898 7.38528 8.78562 7.56016 8.78467 7.73533V10.914H7.218V7.714C7.218 7.54467 7.21533 7.37867 7.206 7.21267C7.20054 7.05547 7.1683 6.90035 7.11067 6.754C7.05629 6.61574 6.95941 6.49832 6.834 6.41867C6.70467 6.33533 6.51667 6.292 6.26467 6.292C6.19067 6.292 6.092 6.308 5.972 6.34133C5.852 6.37533 5.732 6.43667 5.61867 6.52933C5.49214 6.6375 5.39203 6.77319 5.326 6.926C5.246 7.09867 5.206 7.326 5.206 7.606V10.9167H3.64V5.20667H5.11667ZM15.5787 15.6333V0.366667H14.48V0H16V16H14.48V15.6333H15.5787Z' />
+                                </svg>
+                            </a>
+                        </div>
+                    `,
+                },
+            ],
+        },
+        {
+            title: 'Projects',
+            items: [
+                {label: 'go-storage', href: 'https://github.com/beyondstorage/go-storage'},
+                {label: 'go-ftp-server', href: 'https://github.com/beyondstorage/go-ftp-server'},
```

Comment:
Let's remove it from our projects for now.

Reply:
remove go-storage and go-ftp-server ?

Reply:
remove `go-ftp-server` and keep `go-storage`